projects/event-lambdas/src/event-api-lambda/application.ts

```diff
@@ -99,8 +100,33 @@ export const createApp = (initConfig: AppConfig): express.Application => {
           });
           console.log(logJson);
 
+          const viewEvent: IUserTelemetryEvent = {
+            app: lambdaApp,
+            stage: lambdaStage,
```

Comment:
This PR uses the Lambda App and Stage to namespace these tracking events and resolve: https://github.com/guardian/editorial-tools-user-telemetry-service/pull/52#discussion_r1994028727